File: app/src/qaloon/java/com/quran/labs/androidquran/data/QuranFileConstants.java
package com.quran.labs.androidquran.data;

public class QuranFileConstants {
  // server urls
  public static final String BASE_HOST = "http://android.quran.com/data/";
  public static final String IMG_BASE_URL = BASE_HOST + "qaloon/";
  public static final String IMG_ZIP_BASE_URL = IMG_BASE_URL + "zips/";
  public static final String PATCH_ZIP_BASE_URL = IMG_BASE_URL + "patches/v";
  public static final String DATABASE_BASE_URL = BASE_HOST + "databases/";
  public static final String AYAHINFO_BASE_URL = IMG_BASE_URL + "databases/ayahinfo/";

  // local paths
  public static final String QURAN_BASE = "quran_android/";
  public static final String DATABASE_DIRECTORY = "databases";
  public static final String AUDIO_DIRECTORY = "audio";
  public static final String AYAHINFO_DIRECTORY = "qaloon/" + DATABASE_DIRECTORY;
  public static final String IMAGES_DIRECTORY = "qaloon";

  // images version
  public static final int IMAGES_VERSION = 1;
}

File: app/src/qaloon/java/com/quran/labs/androidquran/data/QuranFileConstants.kt
package com.quran.labs.androidquran.data

import com.quran.labs.androidquran.ui.util.TypefaceManager
import com.quran.labs.androidquran.database.DatabaseHandler

object QuranFileConstants {
  // server urls
  const val FONT_TYPE = TypefaceManager.TYPE_UTHMANIC_QALOON

  // arabic database
  const val ARABIC_DATABASE = "quran.ar.qaloon.db"
  const val ARABIC_SHARE_TABLE = DatabaseHandler.ARABIC_TEXT_TABLE
  const val ARABIC_SHARE_TEXT_HAS_BASMALLAH = true
  const val FETCH_QUARTER_NAMES_FROM_DATABASE = false

  const val FALLBACK_PAGE_TYPE = "qaloon"
  const val SEARCH_EXTRA_REPLACEMENTS = ""

  var ICON_RESOURCE_ID = com.quran.labs.androidquran.pages.qaloon.R.drawable.icon
}
